gulpfile.js

```diff
@@ -74,7 +74,20 @@ gulp.task('umd', function(cb) {
         entry: './dist/index.js',
         output: {filename: 'dist/bundles/ng-bootstrap.js', library: 'ngb', libraryTarget: 'umd'},
         devtool: 'source-map',
-        externals: {'@angular/core': ngExternal('core'), '@angular/common': ngExternal('common')}
+        externals: {
+          '@angular/core': ngExternal('core'),
+          '@angular/common': ngExternal('common'),
+          '@angular/forms': ngExternal('forms'),
+          '@angular/http': ngExternal('http'),
```

Comment:
We are not using `http` in the library itself (only in the demo) so it is not strictly needed. But it doesn't hurt either so I'm fine with keeping it.